### PR TITLE
AC Automated Test - LRAC-8892, LRAC-8893 | master

### PR DESCRIPTION
**Pull request by 91nilo**
https://issues.liferay.com/browse/LRAC-8892
https://issues.liferay.com/browse/LRAC-8893

### CODE REVIEW

**Comment by 91nilo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 1]
```diff
@@ -159,18 +159,79 @@ definition {
...
+
+		task ("Unsuppress the user") {
+			Button.click(button = "Unsuppress");
+		}
+
+		task ("Assert that the user was unsuppressed") {
+			ACSettings.createRequestLogByEmail(
+				emailAdresses = "test@liferay.com",
+				jobTypes = "Access");
+		}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:199]: Hi @91nilo, I think to verify that he was unsuppressed, you should go to Request Log and check the list that your user has the type unsuppressed

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 2]
```diff
@@ -159,18 +159,79 @@ definition {
 	@ignore = "true"
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:159]: Hi @91nilo, remove ignore

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 3]
```diff
@@ -159,18 +159,79 @@ definition {
...
+
+		task ("Assert that the user was unsuppressed") {
+			ACSettings.createRequestLogByEmail(
+				emailAdresses = "test@liferay.com",
+				jobTypes = "Access");
+		}
 	}
 
 	@description = "Automation ID: LRAC-8892 | Test Summary: Suppressed user appears in Suppressed User List"
 	@ignore = "true"
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:203]: Hi @91nilo, remove ignore

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase (hunk 1 above)
Comment (91nilo) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:199]: Hi @ViniLopes20, i tought creating a request with the user would be a more practical way to demonstrate that its unsupressed what do you think?

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by 91nilo** [comment]
Fixed. Thanks!

**Comment by alexcandidos** [comment]
Hi @91nilo , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**